src/renderer/pages/common/SortsAndFilters.tsx

```diff
@@ -35,7 +34,7 @@ const optionButtonLike = css`
   margin: 0;
   border: 1px solid ${borderColor};
   border-left: none;
-  color: ${(props) => props.theme.baseText};
+  color: #fffff0;
```

Comment:
Why is this not depending on the theme anymore? It's more an observation, but I found that e.g. in `src\renderer\modal-widgets\PlanInstall\index.tsx` there's still a
```ts
color: ${(props) => props.theme.baseText};
```

Reply:
Hmm, don't know that this was intended.  I've reverted, thank you for pointing this out.

Reply:
@3ter Pushed the final bit into the pull request-had missed the "game management" modal window.  Should be no more changes from here, unless something's off that I didn't see.